### ISSUE TITLE
Fix language selector (Success Criterion 3.1.2: Language of Parts)

### ISSUE THREAD

**Issue by DagA11y** [it-vegard/a11y-reference-website#106]
When on Norwegian site we expose "Choose language: " to screen-reader users - that span should have lang="en" otherwise we fail 3.1.2 in my opinion.

`<span class="visually-hidden">Choose language: </span>` should be `<span lang="en" class="visually-hidden">Choose language: </span>`

There is also a problem with alt text "Norwegian" - that should maybe be "Norsk" then (two instances)

And English flag alt text is also "English" but on a norwegian site...
